### ISSUE TITLE
Resolve broken mailchimp

### ISSUE THREAD

**Issue by arekkas** [serlo-org/athene2#554]
@Knorrke I had to revert your change regarding the new popover: https://github.com/serlo-org/athene2/commit/677b3d9aa675cec9ff389942d90e1680d9a935ff

It caused problems in production, specifically:

```
[Error] TypeError: k.lang is not a function. (In 'k.lang(p)', 'k.lang' is undefined)
    o (scripts.js:33:23289)
    initialize (scripts.js:33:25957)
    (anonymous function) (scripts.js:33:26058)
    l (scripts.js:10:4648)
    fireWith (scripts.js:10:5451)
    ready (scripts.js:9:14063)
    xa (scripts.js:9:11422)
[Error] Failed to load resource: the server responded with a status of 500 (Request Blocked) (analytics.js, line 0)
[Error] Failed to load resource: the server responded with a status of 500 (Request Blocked) (hotjar-306257.js, line 0)
```

Did you test the change on your local machine before making the PR?

**Comment by Knorrke** [comment]
I'll look into it. I did check it before making the PR but maybe I have some local changes that weren't commited, even though I don't think thats the case...

**Comment by Knorrke** [comment]
My bad, sorry.
Tried to resolve the issues, however mailchimp does some strange things with RequireJS, which breaks our script and I couldn't fix that. I will wait until #556 is merged. Most probably this will fix the issue as it gets rid of RequireJS in our script.

**Comment by Knorrke** [comment]
Resolved by new assets #584 :-)
